search/search_integration_tests/smoke_test.cpp

```diff
@@ -223,6 +223,8 @@ UNIT_CLASS_TEST(SmokeTest, CategoriesTest)
       {"highway", "trunk_link"},
       {"highway", "unclassified"},
       {"historic", "citywalls"},
+      {"piste:type", "downhill"},
+      {"piste:type", "nordic"},
```

Comment:
snow_park, playground, skitour, even better - add everything. Do we differentiate them in types? This is important.

Reply:
We don't have minor piste types.

Reply:
@vng those are not minor, they are different ones. There are fewer of them, but they are still very interesting for many skiers. Is it hard to add them? How to do that?